packages/big-design/src/components/ToggleButtons/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { StyledToggleButton } from './styled';
+
+export interface ToggleButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  label: string;
+  selected: boolean;
+}
+
+export const ToggleButton: React.FC<ToggleButtonProps> = React.memo(props => {
```

Comment:
For most/all of our components we want to restrict adding your own custom styling. This should make sure that doesn't happen:
```suggestion
export const ToggleButton: React.FC<ToggleButtonProps> = React.memo(({ className, style, ...props })=> {
```